app/graphql/types/mutation.rb

```diff
@@ -13,6 +13,7 @@ class Mutation < Types::BaseObject
     field :deleteTestResults, mutation: Mutations::TestResult::Delete
     field :associateSystems, mutation: Mutations::Profile::AssociateSystems
     field :associateProfiles, mutation: Mutations::System::AssociateProfiles
+    field :associateRules, mutation: Mutations::Profile::AssociateRules
```

Comment:
New types and mutations need the schema file to be updated. See test failure and/or run `bundle exec rake dump_schema`.